src/main/cpp/t06_longest.cpp

```diff
@@ -17,10 +17,37 @@
 
 #include "t06_longest.h"
 #include <iostream>
+#include <string>
 
-
-using namespace std;
-
+std::string blabla(std::string line){
+    int curr_len = 0;
+    std::string max_word = "";
+    std::string curr_word = "";
+    for(int i =0; i < line.length(); i++){
+        int max_len = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'max_len' can be reduced.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4718331)